src/components/Text/Text.tsx

```diff
@@ -13,8 +13,10 @@ export interface TextProps extends React.HTMLAttributes<HTMLElement> {
    * The OOTB style to apply to the text
    */
   variant?: TextVariants;
-
-  superScript?: string;
+  /**
+   * Text to render special symbols
+   */
+  superScript?: React.ReactNode;
```

Comment:
do we still need this as a property or could it just be passed as one of the children?

Reply:
you right, good catch